scripts/storage-emulator-integration/tests.ts

```diff
@@ -1341,6 +1341,36 @@ describe("Storage emulator", () => {
           })
           .expect(200);
       });
+
+      it("#uploadResumableHandlesAuthError", async () => {
```

Comment:
naming nit: mentioned this in the review for #4214 but could you conform these test case names with the rest of the file (#camelCase -> natural english)

Reply:
Will change this one and fix the rest in a separate PR.